.github/workflows/linting.yml

```diff
@@ -344,7 +344,7 @@ jobs:
     needs: changed_files
     if: needs.changed_files.outputs.js_excluded_files != '[]'
     continue-on-error: true
-    timeout-minutes: 10  # 2021-03-05: Taking ~4:30 now due to now installing all php/js deps to ensure valid linting.
+    timeout-minutes: 10  # 2025-11-06: Takes about a minute, but rarely runs.
```

Comment:
`tools/eslint-excludelist.json` is almost empty now, which explains both the time decrease and the rarely running.